src/tools/mega65_ftp.c

```diff
@@ -383,6 +418,9 @@ int execute_command(char* cmd)
   else if (parse_command(cmd, "put %s %s", src, dst) == 2) {
     upload_file(src, dst);
   }
+  else if (parse_command(cmd, "wput %s", src) == 1) {
```

Comment:
Please rename wput, as wput and wget have the meaning of "wildcard put" and "wildcard get" on many FTP clients, and we may want to implement those (I'd certainly like them :) at some point. Perhaps dput or pput instead?

Reply:
Hmm, I wasn't familiar with an ftp-command called wput. A googling around, I saw mention of a linux-based wput tool, similar to wget. So ok, happy to change the name to disassociate from them a little.

Ok, will change it to `dput`, which people can associate it to mean 'disk put' (i.e., wrap it in a disk and put it).